### PR TITLE
Bz1574264 creating service master

### PR DESCRIPTION
**Pull request by suyogsainkar**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1574264

### CODE REVIEW

File: art-Creating_a_Service_for_VM_Provisioning/topics/Examining_the_Provisioning_Dialog.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[[provisioning-dialog]]
+== Examining the Provisioning Dialog
+
+Understanding provisioning dialogs is important to work with services in {product-title_short}. Every provider in {product-title_short} comes with a sample provisioning dialog covering the options specific to that provider. The process begins with the provisioning dialog, which presents the basic configuration options for a virtual machine or instance. The options presented are dependent on the provider you are using. For instance, a cloud provider might have flavors of instances, whereas an infrastructure provider might allow you to set the memory size or number of CPUs on a virtual machine.
```
Comment (cbudz) [inline art-Creating_a_Service_for_VM_Provisioning/topics/Examining_the_Provisioning_Dialog.adoc]: Suggested revision of the first sentence here: "Understanding provisioning dialogs is important when working with.."

File: art-Creating_a_Service_for_VM_Provisioning/topics/Overview.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[[example-creating-a-service-overview]]
+== Overview
+
+{product-title} provides the ability to define services, which can include one or more virtual machines or instances, and can be deployed across hybrid environments. Services can be made available to users through the Self Service user interface that allows users to order predefined IT services without IT operations getting involved.
```
Comment (cbudz) [inline art-Creating_a_Service_for_VM_Provisioning/topics/Overview.adoc]: Suggested edit: "...through the Self Service user interface, allowing users to order..."

File: art-Creating_a_Service_for_VM_Provisioning/topics/Provisioning_a_VM_Using_SSUI.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+. Log in to the Self Service User Interface at https://<your_cloudforms_appliance>/self_service.
+. Click the *Service Catalog* tab.
+. Click the service you created.
+. In the *Service and VM Name*:
+.. Enter the service name in *Service Name*.
+.. Enter the virtual machine name in *VM Name*.
+.. Enter the virtual machine description in *VM Description*.
+. In *VM Characteristics*, enter *Number of CPUs* and *VM Memory*.
+. Click *Add to Shopping Cart* and access the shopping cart by clicking its icon on the upper-right corner of the screen.
+. Click *Order*. You have ordered a new provisioning request. You can follow its progress and see additional details of the order request by selecting it from the *Orders* tab.
```
Comment (cbudz) [inline art-Creating_a_Service_for_VM_Provisioning/topics/Provisioning_a_VM_Using_SSUI.adoc]: In this last step, once the procedure is ended, any additional 'next steps' should follow in a separate paragraph. I wonder about this so I looked it up in the IBM guide (p89, ending a procedure).

File: art-Creating_a_Service_for_VM_Provisioning/topics/Provisioning_a_VM_Using_SSUI.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+See the link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6/html-single/self_service_user_interface_guide/[Self Service User Interface Guide] for more information about using the Self Service user interface.
+
+
+.Summary
+
+As part of the above sample service, you created a provisioning dialog that defines the options that can be set on a virtual machine and you created a service dialog which allows us to expose certain options to be set by the user. For our example, the service name, virtual machine name, virtual machine description, and virtual machine characteristics are configurable. Then, you created a service catalog and finally a catalog item. The catalog item combines the service dialog with all of the options in the provisioning dialog. Lastly, you provisioned a Red Hat Virtualization virtual machine using the Self Service user interface.
```
Comment (cbudz) [inline art-Creating_a_Service_for_VM_Provisioning/topics/Provisioning_a_VM_Using_SSUI.adoc]: This first sentence can be broken out into two sentences. "Then, you created a service dialog...." Once you discuss the example, I would revise the next sentence to read something like, "Additionally, you created a service dialog..."

**Comment by suyogsainkar** [comment]
@cbudz Hey Chris, thanks for your thorough review and useful suggestions. I've applied all of it in my latest commit, and it's ready for reviewing/merging again. Thanks, Suyog

**Comment by cbudz** [comment]
Hey @suyogsainkar 

Changes look good, thanks for making them. Approved and merging this PR.

-Chris

**Pull request merged**